utils/node_local_test.py

```diff
@@ -909,6 +909,13 @@ def run_tests(dfuse):
     os.symlink(symlink_dest, symlink_name)
     assert symlink_dest == os.readlink(symlink_name)
 
+    # DAOS-6238
+    fname = os.path.join(path, 'test_file4')
+    ofd = os.open(fname, os.O_CREAT | os.O_RDONLY | os.O_EXCL)
+    assert_file_size_fd(ofd, 0)
+    os.close(ofd)
+    os.chmod(fname, stat.S_IRUSR)
```

Comment:
Should we verify that chmod succeeded?
```suggestion
    rc = os.chmod(fname, stat.S_IRUSR)
    assert rc == 0
```

Reply:
os.chmod() will raise an exception of it fails (as it does without the other code changes in this PR) so this isn't needed.

Reply:
Oh, yes, this is python. Sorry 👍